app/controllers/post_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PostController < ApplicationController
```

Comment:
Controller class names use CamelCase and have Controller as a suffix. The Controller suffix is always singular. **The name of the resource is usually plural**.